openedx/core/djangoapps/content/course_overviews/management/commands/generate_course_overview.py

```diff
@@ -0,0 +1,55 @@
+"""
+Command to load course overviews.
+"""
+import logging
+from optparse import make_option
+
+from django.core.management.base import BaseCommand, CommandError
+from opaque_keys.edx.keys import CourseKey
+from xmodule.modulestore.django import modulestore
+
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Example usage:
+        $ ./manage.py lms generate_course_overview --all --settings=devstack
+        $ ./manage.py lms generate_course_overview 'edX/DemoX/Demo_Course' --settings=devstack
+    """
+    args = '<course_id course_id ...>'
+    help = 'Generates and stores course overview for one or more courses.'
+
+    option_list = BaseCommand.option_list + (
+        make_option('--all',
+                    action='store_true',
+                    default=False,
+                    help='Generate course overview for all courses.'),
+    )
+
+    def handle(self, *args, **options):
+
+        if options['all']:
+            course_keys = [course.id for course in modulestore().get_courses()]
+        else:
+            if len(args) < 1:
+                raise CommandError('At least one course or --all must be specified.')
+            course_keys = [CourseKey.from_string(arg) for arg in args]
```

Comment:
Nit: did you want to add a try-except clause here in case an invalid course-key string was passed in?